app/components/Header/Header.jsx

```diff
@@ -3,6 +3,8 @@ import './_Header.scss';
 import AppActions from '../../actions/AppActions';
 import Login from '../Login/Login';
 import page from 'page'; //Note: Can abstract to not rely heavily on page
+import CartStore from '../../stores/CartStore';
```

Comment:
Header can just get the `cartCount` information as a prop from `App.jsx` no need for the whole state of the store or connecting to the store